colorpickerview/src/main/java/com/skydoves/colorpickerview/AlphaTileView.java

```diff
@@ -102,7 +102,7 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
   protected void onDraw(Canvas canvas) {
     super.onDraw(canvas);
     canvas.drawBitmap(backgroundBitmap, 0, 0, null);
-    canvas.drawRect(0, 0, getMeasuredWidth(), getMeasuredHeight(), colorPaint);
+    canvas.drawRect(0, 0, getWidth(), getMeasuredHeight(), colorPaint);
```

Comment:
@Ultron55 Is there any reason to use `getWidth` than `getMeasuredWidth`?

Reply:
The main problem was in the method in which the element was already drawn, so there I used `getWidth()` instead of `getMeasuredWidth` - to exclude inaccuracy. 
Yes, when viewing the changes, I did not pay attention to the fact that I replaced `getMeasuredWidth()` with `getWidth()` in `onLayout` - i could leave it there. 
But I don't think that's a significant problem.